### ISSUE TITLE
Update README to mention how to update an in-progress feature branch from origin/master

### ISSUE THREAD

**Issue by sethladd** [reenhanced/gitreflow#125]
Assuming changes have come into origin/master while I've been working on my local feature branch. What's the git-fu that master reflowers use to update my local feature branch with the latest and greatest?

(apologies if I missed it in the README, I checked and couldn't find it)

I've been doing a standard:

```
(inside of my local feature branch)
git fetch origin
git merge origin/master
```

But I'm sure there's a better reflow-approved way.

Thanks!

**Comment by codenamev** [comment]
`git pull origin master` is what we use, which does the same thing you mentioned.

There is a feature request open (#74) we should be able to get to shortly to make this a reflow command.  `git reflow update` maybe?

**Comment by sethladd** [comment]
That would help. At minimum, some talk about the preferred workflow in the README will help, even if it's `git pull origin master` , which is pretty reasonable.

**Comment by codenamev** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;v4yoxMub&#x2F;22-164-issue-66-74-implementing-git-reflow-refresh"><img src="https:&#x2F;&#x2F;github.trello.services&#x2F;images&#x2F;trello-icon.png" width="12" height="12"> [164] [Issue #66 + #74] Implementing &quot;Git Reflow Refresh&quot;</a>

**Comment by codenamev** [comment]
Fixed in v0.8.0 with 3b44510b7042f7aa6db76248aa52123145732c0a. `git reflow refresh` :smile:
